src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("7.1.2.0")]
-[assembly: AssemblyFileVersion("7.1.2.0")]
+[assembly: AssemblyVersion("8.0.0.0")]
```

Comment:
Why the major version change?  I would suggest mapping the old values Up and Down in custom code to save customers a breaking change.